dali/base/dautils.cpp

```diff
@@ -1479,7 +1479,12 @@ bool CDfsLogicalFileName::getExternalPath(StringBuffer &dir, StringBuffer &tail,
                         *e = makeStringExceptionV(-1, "plane:: does not support planes with more than one device '%s'", planeName.str());
                     return false;
                 }
-                dir.append(plane->queryPrefix());
+                const char * prefix = plane->queryPrefix();
+                //If the prefix is a PathSepChar, it should not be appended to the dir here because
+                //a PathSepChar will be appended to the dir inside the expandExternalPath() if the s
+                //is started with the "::".
+                if (!streq(prefix, iswin ? "\\" : "/"))
```

Comment:
`if (!isRootDirectory(prefix))
`
is better, but will merge as-is

Reply:
I don't think it's because it's "/" per se - I think it's problematic if there's a trailing '/' on the prefix.

Reply:
Yes, I've tested, and if any plane prefix has a trailing '/', the result will be wrong, i.e. it will have a double // at the junction between the end of the prefix and the filePath portion.
So the fix here should be to check for trailing pathsepchar, and avoid appending it to 'dir', that will then also be correct for when prefix == "/"

Reply:
Will change to isRootDirectory() and add the removeTrailingPathSepChar().